src/modals/clargsWizardModal.js

```diff
@@ -39,7 +37,8 @@ const clargsWizardModal = (screen, application) => {
     closeModal();
   });
 
-  return wizardLayout;
+  screen.render();
```

Comment:
I screen render needed here?

Reply:
Necessary as the render loop in Main() hasn't started yet. Can be moved outside of the modal file.